### ISSUE TITLE
merge dreambooth and styledrop

### ISSUE THREAD

**Issue by Laidawang** [zideliu/StyleDrop-PyTorch#21]
<img width="1178" alt="image" src="https://github.com/zideliu/StyleDrop-PyTorch/assets/85244566/0236bbc2-9c4a-4037-8c83-5e512b23cb35">
Is this module included in the code? I can't seem to find it.

**Comment by zideliu** [comment]
No, I didn't do that, I just use style adapter, no object adapter (But, the implementation is same)
